marshmallow_jsonapi/schema.py

```diff
@@ -21,6 +21,14 @@ def plain_function(f):
     return f
 
 
+def dict_items_pop(di):
+    """Pop an item from object returned by dict.items()."""
+    if PY2:
```

Comment:
PY2 support was dropped https://marshmallow-jsonapi.readthedocs.io/en/latest/changelog.html#id2

Reply:
I will check on that. Thanks.